engines/wintermute/base/gfx/opengl/shadow_volume.cpp

```diff
@@ -52,141 +55,144 @@ void ShadowVolume::addVertex(Math::Vector3d &vertex) {
 
 //////////////////////////////////////////////////////////////////////////
 bool ShadowVolume::render() {
-	warning("ShadowVolume::render not implemented yet");
-	//	CBRenderD3D* Rend = (CBRenderD3D*)Game->m_Renderer;
+	glBindTexture(GL_TEXTURE_2D, 0);
+	glBindBuffer(GL_ARRAY_BUFFER, 0);
 
-	//	Rend->m_LastTexture = NULL;
-	//	Rend->m_Device->SetTexture(0, NULL);
+	uint32 *color = new uint32[_vertices.size()];
```

Comment:
Who deletes this? (Also zero-initialize with trailing ())

Reply:
Actually, this is not supposed to be there (used it for some debugging)